app/controllers/DoYouHaveNINController.scala

```diff
@@ -79,9 +79,9 @@ class DoYouHaveNINController @Inject() (
             formWithErrors => render(mode, regime, formWithErrors).map(BadRequest(_)),
             value =>
               for {
-                updatedAnswers <- Future.fromTry(request.userAnswers.set(DoYouHaveNINPage, value))
+                updatedAnswers <- Future.fromTry(request.userAnswers.setOrCleanup(DoYouHaveNINPage, value, true))
```

Comment:
NIN => NINO is this change not happening in this PR

Reply:
This change is not in the main branch ATM.